runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -24,9 +29,18 @@
  * <p>Created by mwei on 12/10/15.
  */
 public class SequencerClient extends AbstractClient {
-
+    private Timer nextTokenTimer;
```

Comment:
please add final modifier for all fields where possible

Reply:
fixed.